### PR TITLE
docs(common):add note for the updated NuGet

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

**Review by svdimitr — APPROVED** [review]

File: _contentTemplates/common/get-started.md [hunk 1]
```diff
@@ -234,3 +234,10 @@ The easiest way to get the commercially licensed Telerik UI for Blazor component
 
 Alternatively, you can also access the `.nupkg` files from [our private NuGet feed]({%slug installation/nuget%}) or by creating a [local feed from your installation]({%slug installation/msi%}#set-up-a-local-nuget-feed-in-visual-studio).
 #end
+
+
+#nuget-update-note
+
+>The improved Telerik NuGet v3 is now available for beta testing at <a href = " https://nuget.telerik.com/v3/index.json" target = "_blank"> https://nuget.telerik.com/v3/index.json. The new v3 API is faster, lighter, and reduces the number of requests from NuGet clients. You are welcome to try it.
```
Comment (marin-bratanov) [inline _contentTemplates/common/get-started.md:241]: Consider reordering the sentences a little - start wit the benefit like "telerik now offers nuget v3 api for our feed which is better, stronger, faster. You can try it at this url".
Consider adding explicit wording around letting us know if there are issues.

File: installation/nuget.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ position: 1
 
 This article explains how to add the private Telerik NuGet feed to your system. You can use it to obtain the Telerik UI for Blazor components.
 
+@[template](/_contentTemplates/common/get-started.md#nuget-update-note)
```
Comment (marin-bratanov) [inline installation/nuget.md:15]: consider adding this template to the nuget feed setup section of the getting started articles (both tutorials and maybe even the "what you need" one).

**Review by marin-bratanov — APPROVED** [review]

File: _contentTemplates/common/get-started.md (hunk 1 above)
Comment (ntacheva) [reply _contentTemplates/common/get-started.md:241]: Done

File: installation/nuget.md (hunk 2 above)
Comment (ntacheva) [reply installation/nuget.md:15]: Done

**Pull request merged**